client/utilities/apiTalk.js

```diff
@@ -123,6 +121,10 @@ export const getResponses = (props, url) => {
 
     const aggregates = makeAggregates(props.questions, props.options, responses);
     props.updateAggregates(aggregates);
+
+    if (url) {
```

Comment:
I thought @chungw51993 had intentionally removed the lines. Jin, can you confirm?